.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+v### Description
+
+[Description of the bug or feature]
+
+### Steps to Reproduce
+
+1. [First Step]
+2. [Second Step]
+3. [and so on...]
+
+### Expected behavior:
+
+[What you expected to happen]
+
+### Actual behavior:
+
+[What actually happened]
+
+### Version of your dependencies, npm and node
+
+[You can get this information from executing the following 3 commands: `npm list --depth=0`, `npm -v` and `node -v`]
```

Comment:
Let's go with '[You can get this information by executing the following command: `npm list --depth=0 && npm -v && node -v`. Just copy and paste it, then hit RETURN in your CLI]'.

Reply:
Done.

Reply:
The 'hit RETURN' don't make sense for me. I think that's better:
[You can get this information by executing the following command: `npm list --depth=0 && npm -v && node -v`. Just copy and paste it]